code/Test_definitions/device-reachability-status.feature

```diff
@@ -13,49 +13,57 @@ Feature: CAMARA Device reachability status API, v0.6.0 - Operations for reachabi
 	
 ############# Happy Path Scenarios ##################	
 
-  @device_reachability_status_01_reachabilityStatusConnectedSms
+  @device_reachability_status_01_reachableAndConnectedSms
   Scenario: Check the reachability status if device is connected with SMS
-    Given a valid devicestatus request body 
+    Given a valid device reachability status request body 
     And the request body property "$.device" is set to a valid testing device which is connected with sms and supported by the service
     When the request "getReachabilityStatus" is sent
     Then the response code is 200
     And the response header "Content-Type" is "application/json"
     And the response header "x-correlator" has same value as the request header "x-correlator"
     And the response body complies with the OAS schema at "/components/schemas/ReachabilityStatusResponse"
-    And the response property "$.status" is 200
-    And the response property "$.code" is "OK"
-    And the response property "$.message" contains a user friendly text
-    And the response property "$.reachabilityStatus" is "CONNECTED_SMS"
+    And the response property "$.reachable" is true
+    And the response property "$.connectivity" is ["SMS"]
```

Comment:
"a valid testing device which is connected with sms and supported by the service" may also be connected for data. So either we specify that the device should also not be connected for data or, better, just check that "SMS" is in the `connectivity` array.
```suggestion
    And the response property "$.connectivity" includes "SMS"
```